### DISCUSSION TITLE
how to use the model to predict AUC, Cmax and Tmax

### DISCUSSION THREAD

**Discussion by LiuYX1998** [Open-Systems-Pharmacology/Forum#1067] [How-Tos]
I have developed a PBPK modeling，and now  I want to know how to use the model to predict AUC, Cmax and Tmax. What should I do next?

**Comment by AndreDlm** [comment]
Run the simulation and click on `Show PK-Analysis` below your chart:
![image](https://user-images.githubusercontent.com/26541603/148185504-eaf69880-5232-4b22-985b-8a13d09f2d82.png)
